petriflow.schema.xsd

```diff
@@ -260,6 +259,7 @@
       <xs:enumeration value="taskRef"/>
       <xs:enumeration value="caseRef"/>
       <xs:enumeration value="filter"/>
+      <xs:enumeration value="i18n"/>

```

Comment:
Please change the issue name and description since it is no longer only about complexType but also merging changes from NAE schema.

Reply:
Done